src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -1822,6 +1831,29 @@ static ov::Shape parseDataShape(const std::string& dataShapeStr) {
     return ov::Shape(dataShape);
 }
 
+std::string getRefBlobFilePath(const std::string& netFileName, const std::vector<std::string>& refFiles,
+                               size_t numberOfTestCase, size_t outputInd) {
+    std::string blobFileFullPath;
+    if (!refFiles.empty() && !FLAGS_ref_dir.empty()) {
+        // Case 1: Reference files & directory are provided (relative path)
+        blobFileFullPath = FLAGS_ref_dir + "/" + refFiles[outputInd];
```

Comment:
nitpick:
I'd rather you employed the same approach with `std::filesystem::path::operator /= (path)` as below than just adding "/" as this sep is not native for Windows

Reply:
Resolved by using platform agnostic `std::filesystem`